clients/src/main/java/org/apache/kafka/clients/consumer/internals/Acknowledgements.java

```diff
@@ -144,6 +144,13 @@ public Acknowledgements merge(Acknowledgements other) {
         return this;
     }
 
+    /**
+     * Returns the Map of Acknowledgements for the offsets.
+     */
+    public Map<Long, AcknowledgeType> getAcknowledgements() {
```

Comment:
Given that this class is called `Acknowledgements`, this is a confusing method name. I suggest `getAcknowledgementTypeMap` or something like that.